gravitee-am-identityprovider/gravitee-am-identityprovider-common/src/main/java/io/gravitee/am/identityprovider/common/oauth2/authentication/AbstractOpenIDConnectAuthenticationProvider.java

```diff
@@ -175,9 +175,15 @@ public Request signInUrl(String redirectUri, String state) {
                     if (httpResponse.statusCode() != 200) {
                         throw new BadCredentialsException(httpResponse.statusMessage());
                     }
-
                     JsonObject response = httpResponse.bodyAsJsonObject();
                     String accessToken = response.getString(ACCESS_TOKEN_PARAMETER);
+                    // We store the token is option is enabled
```

Comment:
We should also do this at line 140, if the response type is "id_token token"

Reply:
For the `op_access_token ` filtered, you want me to do it here ? https://github.com/gravitee-io/gravitee-access-management/pull/1480

Reply:
Done